src/core/core.plugins.js

```diff
@@ -179,204 +179,204 @@ module.exports = {
  * @param {Chart.Controller} chart - The chart instance.
- * @param {Number} easingValue - The current animation value, between 0.0 and 1.0.
- * @param {Object} options - The plugin options.
+ * @param {number} easingValue - The current animation value, between 0.0 and 1.0.
+ * @param {object} options - The plugin options.
  */
 /**
  * @method IPlugin#beforeDatasetDraw
  * @desc Called before drawing the `chart` dataset at the given `args.index` (datasets
  * are drawn in the reverse order). If any plugin returns `false`, the datasets drawing
  * is cancelled until another `render` is triggered.
  * @param {Chart} chart - The chart instance.
- * @param {Object} args - The call arguments.
- * @param {Number} args.index - The dataset index.
- * @param {Object} args.meta - The dataset metadata.
- * @param {Number} args.easingValue - The current animation value, between 0.0 and 1.0.
- * @param {Object} options - The plugin options.
- * @returns {Boolean} `false` to cancel the chart datasets drawing.
+ * @param {object} args - The call arguments.
+ * @param {number} args.index - The dataset index.
+ * @param {object} args.meta - The dataset metadata.
+ * @param {number} args.easingValue - The current animation value, between 0.0 and 1.0.
+ * @param {object} options - The plugin options.
+ * @returns {boolean} `false` to cancel the chart datasets drawing.
  */
 /**
  * @method IPlugin#afterDatasetDraw
  * @desc Called after the `chart` datasets at the given `args.index` have been drawn
  * (datasets are drawn in the reverse order). Note that this hook will not be called
  * if the datasets drawing has been previously cancelled.
  * @param {Chart} chart - The chart instance.
- * @param {Object} args - The call arguments.
- * @param {Number} args.index - The dataset index.
- * @param {Object} args.meta - The dataset metadata.
- * @param {Number} args.easingValue - The current animation value, between 0.0 and 1.0.
- * @param {Object} options - The plugin options.
+ * @param {object} args - The call arguments.
+ * @param {number} args.index - The dataset index.
+ * @param {object} args.meta - The dataset metadata.
+ * @param {number} args.easingValue - The current animation value, between 0.0 and 1.0.
+ * @param {object} options - The plugin options.
  */
 /**
  * @method IPlugin#beforeTooltipDraw
  * @desc Called before drawing the `tooltip`. If any plugin returns `false`,
  * the tooltip drawing is cancelled until another `render` is triggered.
  * @param {Chart} chart - The chart instance.
- * @param {Object} args - The call arguments.
- * @param {Object} args.tooltip - The tooltip.
- * @param {Number} args.easingValue - The current animation value, between 0.0 and 1.0.
- * @param {Object} options - The plugin options.
- * @returns {Boolean} `false` to cancel the chart tooltip drawing.
+ * @param {object} args - The call arguments.
+ * @param {object} args.tooltip - The tooltip.
```

Comment:
This should be `{Tooltip}`?

Reply:
done. I really wasn't trying to fix every possible doc issue in this PR. I only reviewed the capitalization